sdk/nodejs/tests/runtime/testdata/closure-tests/test.ts

```diff
@@ -39,6 +40,17 @@ async function getSnapshot(testCase: string, typescriptVersion: string): Promise
     return fs.readFile(path.join("cases", testCase, "snapshot.txt"), "utf-8");
 }
 
+// This test validates that the typescript version used by the closure tests
+// is the same as the one used by the pulumi package and that we are testing
+// what we think we are testing ...
+it(`resolve to the correct typescript version within the pulumi package`,
```

Comment:
Turns out I was running the tests for a while where we had `node_modules/typescript` at the expected version, but also `node_modules/@pulumi/pulumi/node_modules/typescript` on 3.8.3 ... This seems somewhat easy to break, so this tests ensures we have the correct test setup.